lib/matplotlib/backends/backend_ps.py

```diff
@@ -1440,27 +1440,32 @@ def convert_psfrags(tmpfile, psfrags, font_preamble, custom_preamble,
     command = '%s cd "%s" && latex -interaction=nonstopmode "%s" > "%s"'\
```

Comment:
You've already removed it (e.g., [here](https://github.com/matplotlib/matplotlib/pull/7572/files#diff-d593c407e3c1e32bc40e3d613669377cL401)).